### PR TITLE
Ensure .gitignore gets copied when running `yo @plone/volto-addon`

### PR DESCRIPTION
**Pull request by instification**
Minor bugfix. When running `yo @plone/volto-addon` the `.gitignore` file does not get copied.

I tried to use the suggestions in https://github.com/yeoman/generator/issues/812 but adding `.npmignore` did not solve the problem.

However, iteems like it was already fixed for the app generator: 

https://github.com/plone/volto/blob/afadd4b3b8f4c69f4cef705430d28cb56f0c7b1f/packages/generator-volto/generators/app/index.js#L230-L243

So this PR is porting that fix to the addon generator.

### CODE REVIEW

File: packages/generator-volto/generators/addon/index.js [hunk 1]
```diff
@@ -207,11 +207,6 @@ Run "npm install -g @plone/generator-volto" to update.`,
   }
 
   install() {
-    this.renderTemplate(
-      `${this.templatePath()}/**/*`,
-      this.destinationPath(),
-      this.globals,
-    );
```
Comment (davisagli) [inline packages/generator-volto/generators/addon/index.js:214]: @instification I think this means that `<%= addonName %>` will no longer be interpolated in Makefile.

**Review by davisagli — CHANGES_REQUESTED** [review]

File: packages/generator-volto/generators/addon/index.js (hunk 1 above)
Comment (instification) [reply packages/generator-volto/generators/addon/index.js:214]: Thanks! I've refactored to keep the globals so I think this is going to work ok now.

**Review by davisagli — APPROVED** [review]
LGTM now

**Review by tiberiuichim — APPROVED** [review]
It would be good to have a comment in the code for this, to explain the WTF for the next developer that looks at the code.

**Pull request merged**